src/Microsoft.Azure.WebJobs.Host/Config/ExtensionConfigContextExtensions.cs

```diff
@@ -14,64 +13,6 @@ namespace Microsoft.Azure.WebJobs.Host.Config
     [Obsolete("Not ready for public consumption.")]
```

Comment:
Did you mean to remove this?

Reply:
Not yet - I think we can remove this entire class once we do the DI work. The remaining methods are just about config things.

Reply:
Sounds good.